house/tests/house_login_page_tests.py

```diff
@@ -0,0 +1,70 @@
+import pytest
+from house.constants import LOGIN_PAGE_ROUTE
+from house.views import house_login
+from house.models import House
+from factories.house import HouseFactory
+from django.test.client import RequestFactory
+from house.forms import HouseIDForm
+from django.template.loader import get_template
+from django.template import TemplateDoesNotExist
+
+
+# Fixures:
+
+@pytest.fixture
+def request_factory():
```

Comment:
Why not reuse the fixture and create one for all the tests?

Reply:
> Why not reuse the fixture and create one for all the tests?

Added a common fixture file, and removed all of her shows.
I did it in a different commit so it would be visible 👍